<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>initWindow</class>
 <widget class="QMainWindow" name="initWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>167</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>643</width>
    <height>167</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>643</width>
    <height>167</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="grpUpload">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>601</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Upload csv file</string>
    </property>
    <widget class="QRadioButton" name="rdbtnFile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>File</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdbtnURL">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>URL</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtFile">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnUpload">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>20</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Upload csv</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnFileCSV">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>50</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Buscar</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>643</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
